VAMobile/src/styles/themes/standardTheme.ts

```diff
@@ -48,6 +52,10 @@ const fontSizes = {
     fontSize: 16,
     lineHeight: 20,
   },
+  AnnouncementBannerTitle: {
+    fontSize: 16,
+    lineHeight: 20,
```

Comment:
```suggestion
    lineHeight: 22,
```
Based on the design in Figma